src/test/java/core/basesyntax/report/ReportCreatorImplTest.java

```diff
@@ -0,0 +1,77 @@
+package core.basesyntax.report;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.database.Storage;
+import core.basesyntax.dataio.CsvToStorage;
+import core.basesyntax.dataio.FileReaderImpl;
+import core.basesyntax.dataio.FileWriter;
+import core.basesyntax.dataio.FileWriterImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.TreeMap;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ReportCreatorImplTest {
+    private static ReportCreator reportCreator = new ReportCreatorImpl();
+
+    @Before
+    public void setUp() {
+        Storage.setFruitsMap(new TreeMap<>());
+    }
+
+    @Test
+    public void test1_ReportCreatorOk() {
+        new CsvToStorage(new FileReaderImpl("src/main/resources/test1.cvs").read());
+        String actual = reportCreator.createReport();
+        String expected = "fruit,quantity" + System.lineSeparator()
+                + "apple,20" + System.lineSeparator() + "banana,95";
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void test2_ReportCreatorToFileOk() {
+        new CsvToStorage(new FileReaderImpl("src/main/resources/test2.csv").read());
+        FileWriter fileWriter = new FileWriterImpl();
+        fileWriter.writeToFile(reportCreator.createReport(),
+                "src/main/resources/test2_output.csv");
```

Comment:
the work of the fileWriter should be tested in a separate class - and you won't need to create the report for it, you can just write some random string